### ISSUE TITLE
Anti-drop implant: items can get permenantly stuck in your hand

### ISSUE THREAD

**Issue by SteelSlayer** [ParadiseSS13/Paradise#11526]
**Problem Description**:
If you are using an anti-drop implant that is activated with an object in your hand, and EMP goes off, it causes you to drop the item. If you try to pick up the item you dropped **without first clicking the implant icon to relax your hand**, that item be permanently stuck. In this state, if you try activating the implant with the icon, the game will repeatedly say "Your hands relax.." over and over, but the item will remain stuck.

**What did you expect to happen**: 
I expected your hand to actually relax, allowing you to drop the item.

**What happened instead**:
The item becomes permanently stuck in your hand.

**Why is this bad/What are the consequences:**
If you want to be able to use your hand again, you have seek an admin's help to fix it.

**Steps to reproduce the problem**:
1. Equip yourself with the anti-drop implant
2. Activate the implant while holding an item
3. Trigger an EMP. The gripped item will fly away
4. Pick up the same item you dropped and activate the implant.
5. The item is now stuck in your hand permanently. Activating the implant again will only give the   message "Your hands relax.."

**When did the problem start happening**:
Was able to recreate this bug using a private server using code downloaded from the github on May 12th, 2019. It was reported in-game by a player on the main server on May 13th.

**Comment by Ralta** [comment]
I've posted this before in #8178 and there's an open issue #7300 as well.

**Comment by SteelSlayer** [comment]
Yep, I saw those too. I asked in-game, and admins and other mentors said I should make a new issue so it can get some more attention.

**Comment by farie82** [comment]
dropped is most likely not called by the emp_act when you throw the weapon away
